tests/impl.cpp

```diff
@@ -6081,7 +6081,17 @@ result_t test_mm_stream_si32(const SSE2NEONTestImpl &impl, uint32_t iter)
 
 result_t test_mm_stream_si64(const SSE2NEONTestImpl &impl, uint32_t iter)
 {
-    return TEST_UNIMPL;
+    const int64_t a = (const int64_t) impl.mTestInts[iter];
+#if defined(__x86_64__) || defined(__i386__)
+    // Microsoft Windows (x86-64) takes LLP64
```

Comment:
The comment was misleading. Can you specify with macro `__LP64__`?

Reply:
I have removed the comments, and add macro `__LLP64__` in the `_x86_64` case